docs/installation_guide.rst

```diff
@@ -276,7 +276,7 @@ If you prefer to use docker compose, a docker-compose.yml template is provided b
 Using docker over a public network
 ----------------------------------
 
-Since rotki was designed to run over a public network it is :ref:`generally advised <docker warning>` to avoid
+Since rotki was designed to run over a private network it is :ref:`generally advised <docker warning>` to avoid
```

Comment:
Hahaha someone forgot to put the **not** there.

Thanks for noticing!

Can you perhaps change it to just have a not? I know it's kind of the same thing, but I would prefer it as a wording.

```suggestion
Since rotki was not designed to run over a public network it is :ref:`generally advised <docker warning>` to avoid
```

Reply:
Updated!